docs: add forking-renovate to list of distributions (#13535)
renovatebot/renovate: docs/usage/getting-started/running.md

### Available distributions

#### Forking Renovate app

"Forking Renovate" is the sister app to Renovate Bot.

> The difference is that Forking Renovate operates using forks of repositories instead of as a collaborator, so needs only read-only permissions rather than read-write.

Author: "operates using forks of repositories instead of as a collaborator" this is a really weird sentence, do you have a better wording?

Probably a good idea to change the wording on the Forking Renovate app landing page as well, because that's the source of this text. 😉

It works on public (e.g. open source) repositories only.

[Install Forking Renovate from GitHub App](https://github.com/apps/forking-renovate).

##### Benefits

> Forking Renovate needs only `read` level access to any repository it runs on.

Author: I don't know if there are more benefits than just "only needs `read` level access"?

##### Drawbacks

If you use Forking Renovate, you'll miss out on these features of the regular Renovate app:

- Dependency Dashboard (probably needs `write` rights to be able to open the issue?)
- Automerge with the `branch` automerge strategy (needs `write` rights to create a branch in the first place)

> - List more big ticket items here

Author: Please check if I'm correct in listing the Dependency Dashboard and automerge with `branch` strategy as things you'll miss out on if you use the Forking Renovate app.

We probably want to nudge/steer our users towards the full-fat app if possible?

### Available distributions

> #### Forking Renovate app

Reviewer: This should not be first. Maybe even last

> "Forking Renovate" is the sister app to Renovate Bot.

Reviewer: ```suggestion
"Forking Renovate" is the sister app to the WhiteSource Renovate App on GitHub.com.
```

> The difference is that Forking Renovate operates using forks of repositories instead of as a collaborator, so needs only read-only permissions rather than read-write.

Reviewer: ```suggestion
The difference is that Forking Renovate does not require write permissions to create branches within the repo, and instead submits PRs from its own fork.
```

> It works on public (e.g. open source) repositories only.

Reviewer: ```suggestion
Because of how it works, it functions on public repositories only and additionally cannot support `automerge` capabilities.
```

[Install Forking Renovate from GitHub App](https://github.com/apps/forking-renovate).

##### Benefits

Forking Renovate needs only `read` level access to any repository it runs on.

##### Drawbacks

If you use Forking Renovate, you'll miss out on these features of the regular Renovate app:

> - Dependency Dashboard (probably needs `write` rights to be able to open the issue?)
> - Automerge with the `branch` automerge strategy (needs `write` rights to create a branch in the first place)
> - The `baseBranches` config option is not supported
> - List more big ticket items here

Reviewer: ```suggestion
- Automerge is not supported
- The `baseBranches` config option is not supported
- The app dashboard (`app.renovatebot.com`) is currently not supported
```
